runtime/onert/core/include/exec/Executors.h

```diff
@@ -18,19 +18,34 @@
 #define __ONERT_EXEC_EXECUTORS_H__
 
 #include "IExecutor.h"
+#include "ir/NNPkg.h"
 
 namespace onert
 {
 namespace exec
 {
 
+/**
+ * @brief Struct to gather model I/O information in multimodel NN package
+ *        Model I/O will have role one of below
+ *        - Package input/output
+ *        - Edge's start/finish point between model
+ */
+struct ModelEdges
+{
+  std::vector<ir::IODesc> pkg_inputs;
+  std::vector<ir::IODesc> pkg_outputs;
+  std::unordered_set<ir::ModelEdge, ir::ModelEdgeHash, ir::ModelEdgeEqual> edges;
+};
```

Comment:
@hseok-oh It seems same to members in `NNPkg.h`. What about define it in `NNPkg.h` and reuse it here?

Reply:
OK. I'll update

Reply:
I'll make new PR for this.